cli/src/main/java/io/github/bmarwell/jfmt/JFmt.java

```diff
@@ -1,17 +1,17 @@
-package io.github.bmarwell.jdtfmt;
+package io.github.bmarwell.jfmt;
 
-import io.github.bmarwell.jdtfmt.commands.AbstractCommand;
-import io.github.bmarwell.jdtfmt.commands.Diff;
-import io.github.bmarwell.jdtfmt.commands.List;
-import io.github.bmarwell.jdtfmt.commands.Print;
-import io.github.bmarwell.jdtfmt.commands.Write;
+import io.github.bmarwell.jfmt.commands.AbstractCommand;
+import io.github.bmarwell.jfmt.commands.Diff;
+import io.github.bmarwell.jfmt.commands.List;
+import io.github.bmarwell.jfmt.commands.Print;
+import io.github.bmarwell.jfmt.commands.Write;
 import picocli.CommandLine;
 import picocli.jansi.graalvm.AnsiConsole;
 
 @CommandLine.Command(
-    name = "jdtfmt",
+    name = "jfmt",
     mixinStandardHelpOptions = true,
-    version = "jdtfmt 1.0",
+    version = "jfmt 1.0",
```

Comment:
Unrelated nit: rather than hardcoding the version here, you can have it resolved dynamically at runtime. See [how I did it in MCS](https://github.com/mthmulders/mcs/blob/main/src/main/java/it/mulders/mcs/cli/ClasspathVersionProvider.java) to have an idea. I think other approaches (maybe based on the **META-INF/MANIFEST.MF** are also possible.

Reply:
Yep, let's make this a new issue!